docs/examples/cas9_kymotracking/cas9_kymotracking.rst

```diff
@@ -137,17 +137,18 @@ bead. This binding should be omitted from the analysis::
 One last thing to note is that we assigned the `KymoWidgetGreedy` to the variable `kymowidget`. That means that from
 this point on, we can interact with it through the handle name `kymowidget`.
 
-Exporting from the widget results in a file that contains the line coordinates in pixels and real units.
-If we also want to export the photon counts in a region around the traced line, we can include a `sampling_width`.
+Exporting from the widget results in a file that contains the track coordinates in pixels and real units.
+If we also want to export the photon counts in a region around the track, we can include a `sampling_width`.
 This sums the photon counts from `pixel_position - sampling_width` to (and including) `pixel_position + sampling_width`::
 
     kymowidget.save_lines("kymotracks_calibrated.txt", sampling_width=3)
 
 Analyzing the results
 ---------------------
 
-Once traced, the lines are available in `kymowidget.lines`. Lines have a `position` list and a `time` list. Let’s grab
-the longest line we found, and have a look at its position over time::
+The tracks are available in `kymowidget.lines`, which returns a `KymoTrackGroup` object. The group is a customized list of `KymoTrack` objects
```

Comment:
Hm, is it really `kyowidget.lines`? Was this what you mentioned about additional necessary changes?

Reply:
this will be changed in another PR fixing up `kymotracker_widgets.py`; for this PR I focused solely on the former `kymolines.py`